apps/heft/src/cli/HeftActionRunner.ts

```diff
@@ -453,6 +453,7 @@ function _getOrCreatePhaseOperation(
     // Only create the operation. Dependencies are hooked up separately
     operation = new Operation({
       groupName: phase.phaseName,
+      name: `${phase.phaseName} phase`,
```

Comment:
What is the appropriate name for a phase?

In the source code, there currently seem to be very few places that actually read the `name` or `groupName`.

Reply:
A phase operation is setup work that runs before all tasks in the given phase, so it should indicate something about setup or initialization.

Reply:
Currently the phase operation is only used for performing the clean.